### PR TITLE
Fixes #23924: The technique is no longer working on Rudder 8.0+

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/23924

### CODE REVIEW

**Comment by Fdall** [comment]
Commit modified

**Comment by Fdall** [comment]
PR updated with a new commit

File: openscap/README.adoc [hunk 1]
```diff
@@ -17,7 +17,7 @@ The plugin provides a Technique to execute the OpenSCAP tool at regular interval
 == Installation
 
 * Your Rudder server must have python-requests and rudder-api-client installed
-* Install the plugin on the Rudder Server with `rudder package`
+* Install the plugin on the Rudder Server with `rudder package install rudder-plugin-openscap`
```
Comment (amousset) [inline openscap/README.adoc:20]:
```suggestion
* Install the plugin on the Rudder Server with `rudder package install openscap`
```

**Review by amousset — APPROVED** [review]

**Comment by Fdall** [comment]
OK, squash merging this PR

**Pull request merged**